plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2933,3 +2933,42 @@ function wc_update_950_tracking_option_autoload() {
 	);
 	wp_set_option_autoload_values( $options );
 }
+
+/**
+ * Modifies PKs for woocommerce_order_itemmeta and wp_wc_orders_meta tables for more robust select operations.
+ */
+function wc_update_970_modify_primary_key_for_order_related_meta_tables() {
+	global $wpdb;
+
+	$prefix  = $wpdb->prefix;
+	$changes = array(
+		// NEXT: contradicts with the function name a bit.
+		'options'                    => array(
+			'original_pk' => 'option_id',
+			'query'       => "ALTER TABLE {$prefix}options ADD UNIQUE KEY option_id (option_id), DROP PRIMARY KEY, ADD PRIMARY KEY (option_name), DROP KEY option_name",
+		),
+		'woocommerce_order_itemmeta' => array(
+			'original_pk' => 'meta_id',
+			'query'       => "ALTER TABLE {$prefix}woocommerce_order_itemmeta ADD UNIQUE KEY meta_id (meta_id), DROP PRIMARY KEY, ADD PRIMARY KEY (order_item_id, meta_key, meta_id), DROP KEY order_item_id",
+		),
+		'woocommerce_payment_tokenmeta' => array(
```

Comment:
Benchmarking (50000 x 10 entries in the table):
Insert: 0.000329 -> 0.000290  seconds / query
Select: 0.000250 -> 0.000268 seconds / query

Not much value either.